pkg/kubelet/cm/topologymanager/socketmask/socketmask_test.go

```diff
@@ -253,7 +253,7 @@ func TestCount(t *testing.T) {
 	}{
 		{
 			name:          "Count number of bits set in full mask",
-			maskBit:       0,
+			maskBit:       42,
```

Comment:
I'm having a little trouble understanding how the previous version of this test passed? It seems like if the mask bit is 0, then there should be no 1's right? similarly, if the mask bit is 42, shouldn't the count of 1s be more than 1?

Reply:
The previous test ssets bit 0 in the mask, so 1 bit is set. The new test just changes this to bit 42. There is no real reason to pick bit 42 over bit 0. It doesn't change the outcome.

Reply:
@mattjmcnaughton previous version of the test sets first bit. For function that supposed to count amount of 1s in the bitmask, is a very trivial case. It would be of course better to have test completely rewritten to have it tested 2-3 bits set in the beginning/middle/end of the range... But well, I think checking that mask has 1 bit set somewhere in the middle is good enough for updated code that uses standard library.